client/js/utils/communication_handler.js

```diff
@@ -5,8 +5,6 @@ export default class {
 
   constructor(){
     Communicator.enableListener(this);
-    this.prevHeight = this.getHeight();
```

Comment:
If these are set in the constructor then a message won't be sent until there is a size change from the initial size